tools/testing/target_determination/heuristics/utils.py

```diff
@@ -0,0 +1,41 @@
+import os
+import subprocess
+from pathlib import Path
+from typing import List, Set
+
+REPO_ROOT = Path(__file__).resolve().parent.parent.parent.parent.parent
```

Comment:
This line is so lol

Reply:
🫣 (this could really use a more generic helper function...)